samples/scaleout/distributor-upgrade/Core_6/Worker1/Program.cs

```diff
@@ -23,7 +24,7 @@ static void Main()
         endpointConfiguration.EnlistWithLegacyMSMQDistributor(
             masterNodeAddress: ConfigurationManager.AppSettings["DistributorAddress"],
             masterNodeControlAddress: ConfigurationManager.AppSettings["DistributorControlAddress"],
-            capacity: 10);
+            capacity: PushRuntimeSettings.Default.MaxConcurrency);
```

Comment:
I wondered what `PushRuntimeSettings` is, I searched docs but it is only used in a few samples.